### ISSUE TITLE
This project is continuing update?

### ISSUE THREAD

**Issue by skeleton1231** [floatinghotpot/casino-server#23]
I plan to use this framework to develop on our product, it is ok?  Thanks for suggest

**Comment by floatinghotpot** [comment]
Sure, it's okay. 

The architecture itself is stable, and based on it, more game rules and client API can be added.

**Comment by skeleton1231** [comment]
Is there plan to develop Fight Landlord and Blackjack? 
and the TODO List ?

**Comment by floatinghotpot** [comment]
Yes, we have the plan, but before that, we are focusing on developing a working Unity client.

**Comment by haoxingfu** [comment]
When will the unity client publish?

**Comment by remadrivan** [comment]
It depends. If Raymond will update

**Comment by alexey-sh** [comment]
What about roulette?

**Comment by madgreg99** [comment]
Hello, nice work! I'm wondering the same thing as @haoxingfu about Unity, is this still in development?
